apps/system/test/unit/app_install_manager_test.js

```diff
@@ -683,8 +684,8 @@ suite('system/AppInstallManager >', function() {
           setup(function() {
             // reseting these mocks as we want to test only one call
             MockNotificationScreen.mTeardown();
-            MockStatusBar.mTeardown();
-
+            Service.request.restore();
```

Comment:
If my memory goes well, removing the restore will cause the test to run twice on its own and the second run will fail. It looks like the appInstallManager has a nested test route and if we don't clear the stub, you will "TypeError: Attempted to wrap request which is already stubbed"